parser-typechecker/src/Unison/Codebase/Editor/Command.hs

```diff
@@ -190,3 +223,52 @@ data Command m i v a where
 
   RuntimeMain :: Command m i v (Type v Ann)
   RuntimeTest :: Command m i v (Type v Ann)
+
+commandName :: Command m i v a -> String
```

Comment:
Purely a debug tool. This is useful in case you want to print out a trace of all the commands.